scripts/package_skyrim_pdbs.py

```
@@ -0,0 +1,160 @@
+    if os.path.isdir(stage):
+        shutil.rmtree(stage)
+    os.makedirs(plugin_dir, exist_ok=True)
+    staged_pdb = os.path.join(plugin_dir, cfg["consumer_name"])
+    shutil.copy2(src, staged_pdb)
+
+    # Archive: out/<nexus_name>.7z containing SKSE/Plugins/<consumer_name>.pdb
+    archive = os.path.join(out_dir, cfg["nexus_name"] + ".7z")
+    if os.path.isfile(archive):
+        os.remove(archive)
+    # add from the stage root so the archive contains SKSE/Plugins/... (no extra leading dir)
+    proc = subprocess.run(
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)